pkg/domain/filters/volumes.go

```diff
@@ -81,3 +81,21 @@ func GenerateVolumeFilters(filters url.Values) ([]libpod.VolumeFilter, error) {
 	}
 	return vf, nil
 }
+
+func GeneratePruneVolumeFilters(filters url.Values) ([]libpod.VolumeFilter, error) {
+	var vf []libpod.VolumeFilter
+	for filter, v := range filters {
+		for _, val := range v {
+			switch filter {
+			case "label":
+				filter := val
+				vf = append(vf, func(v *libpod.Volume) bool {
+					return util.MatchLabelFilters([]string{filter}, v.Labels())
+				})
+			default:
+				return nil, errors.Errorf("%q is in an invalid volume filter", filter)
```

Comment:
Small suggestion: "%q is an invalid volume prune filter"

Reply:
Nicely caught. I corrected this in the original filter function and here as well.